pythonie/meetups/test_meetups.py

```diff
@@ -13,11 +13,12 @@
 from meetups.models import Meetup, next_n_months
 from meetups.utils import update_needed
 
-description = '<p>We will be having a meetup in June. More details to follow.</p> <p>If you are ' \
-              'interested in speaking, please submit your details to\xa0<a ' \
-              'href="http://bit.ly/pyie-cfp-2015"><a href="http://bit.ly/pyie-cfp-2015" ' \
-              'class="linkified">http://bit.ly/pyie-cfp-2015</a></a>.</p> <p>Enquiries? Please ' \
-              'contact contact@python.ie.</p>'
+description = ('<p>We will be having a meetup in June. More details to follow.'
+               '</p> <p>If you are interested in speaking, please submit your '
+               'details to\xa0<a href="http://bit.ly/pyie-cfp-2015">'
+               '<a href="http://bit.ly/pyie-cfp-2015" class="linkified">'
+               'http://bit.ly/pyie-cfp-2015</a></a>.</p> <p>Enquiries? Please '
+               'contact contact@python.ie.</p>')
```

Comment:
Hi @nicolasunravel , so remove the outer 'a' tag?

+ description = ('<p>We will be having a meetup in June. More details to follow.'
+               '</p> <p>If you are interested in speaking, please submit your '
+               'details to: <a href="http://bit.ly/pyie-cfp-2015" class="linkified">'
+               'http://bit.ly/pyie-cfp-2015</a>.</p> <p>Enquiries? Please '
+               'contact contact@python.ie.</p>')